src/status_im/ui/screens/home/views.cljs

```diff
@@ -96,7 +98,29 @@
                        (when-not search-filter
                          (re-frame/dispatch [:search/home-filter-changed ""])))
      :on-change      (fn [text]
-                       (re-frame/dispatch [:search/home-filter-changed text]))}]])
+                       (re-frame/dispatch [:search/home-filter-changed text])
+                       (when-not (seq chats)
+                         (debounce/debounce-and-dispatch [:new-chat/set-new-identity text] 300)))}]])
+
+(defn start-suggestion [search-value]
+  (let [{:keys [state ens-name public-key]} @(re-frame/subscribe [:contacts/new-identity])]
+    [react/view {}
```

Comment:
**just** wondering, why an empty map is needed here?

Reply:
Yes, it is not needed here